test/WebSites/BasicWebSite/wwwroot/javascript/jquery.validate.unobtrusive.3.2.6.js

```diff
@@ -0,0 +1,416 @@
+/*!
```

Comment:
@Eilon no the filename isn't normally versioned but URLs would be. Without this, we'd have no visible information about where this file came from.

We specifically updated the projects in this repo to _remove_ use of `npm`, `bower`, `grunt`, and `gulp`. I am trying to avoid reversing that decision.